app/layout.module.scss

```diff
@@ -1,36 +1,34 @@
-.body {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  color: #ffffff;
-  height: 100vh;
-}
+@use 'sass:map';
+@import './variables';
 
-.nav {
+.header {
   display: flex;
-  justify-content: space-between;
   align-items: center;
-  padding: 0 20%;
+  justify-content: space-between;
   width: 100%;
   height: 80px;
   backdrop-filter: blur(10px);
 
   z-index: 100;
   position: fixed;
 
-  &_item {
-    display: flex;
-    align-items: center;
-    gap: 2rem;
-  }
-
   &_buttons {
     display: flex;
     gap: 1rem;
   }
+
+  @media (max-width: map.get($grid-breakpoints, 'lg')) {
```

Comment:
media query not on top level it is good practise?

Reply:
it depends, bro =)
in this case i use it for more specific rule for header =)

That's what the AI answered))

```
Best Practice:
Consider adopting a hybrid approach that strikes a balance between the two. Keep simple and generic media queries at the top level, and for more complex or specific cases, consider nesting them within relevant CSS rules.

Additionally, using a CSS preprocessor like Sass or LESS can provide more organization and maintainability through the use of mixins or functions for media queries.

Ultimately, consistency and readability should be prioritized. Choose an approach that makes your CSS codebase maintainable and easy for you and your team to work with.Best Practice:
Consider adopting a hybrid approach that strikes a balance between the two. Keep simple and generic media queries at the top level, and for more complex or specific cases, consider nesting them within relevant CSS rules.

Additionally, using a CSS preprocessor like Sass or LESS can provide more organization and maintainability through the use of mixins or functions for media queries.

Ultimately, consistency and readability should be prioritized. Choose an approach that makes your CSS codebase maintainable and easy for you and your team to work with.
```